bootstrap/kubeadm/main.go

```diff
@@ -14,6 +14,7 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
+// main is the main package for Kubeadm Bootstrap.
```

Comment:
```suggestion
// main is the main package for the Kubeadm Bootstrap controller.
```
nit (please also check the other controller package comments)